versions.def

```diff
@@ -71,11 +71,11 @@ HTTPD_VERSION		:= 2.4.53
 
 IMGK_VERSION		:= 6.7.7-2
 
-JDK_SUB_TREE		:= 8u
-JDK_UPDATE		:= 172
-JDK_BUILD		:= b01
-JDK_TAG			:= jdk8u$(JDK_UPDATE)-$(JDK_BUILD)
-JDK_VERSION		:= 1.8.0u$(JDK_UPDATE)$(JDK_BUILD)
+JDK_SUB_TREE		:= 17
```

Comment:
also JDK 18 is out, any reason to not pick the latest one?
I think it's too late to ask this question now, so if we think we can still go to JDK 18 then let's file separate ticket to upgrade that in future

Reply:
JDK 17 is an LTS release, supported until 2029, whereas 18 is a regular release, only supported until 19 comes out.